src/vector/feature.rs

```diff
@@ -87,4 +150,12 @@ impl FieldValue {
             _ => panic!("not a RealValue")
         }
     }
+
+    /// Interpret the value as `i32`. Panics if the value is something else.
+    pub fn as_int(self) -> i32 {
```

Comment:
i guess this would be the right thing to do. Also i think these methods should return an Option instead of panicking...